src/ziggurat/resource/dead_set.clj

```diff
@@ -20,11 +20,10 @@
     (Integer/parseInt count)
     (catch NumberFormatException ex
       (log/errorf "count %s is not an integer" count)
-      nil)))
+      (str "Count %s is not an integer" count))))
```

Comment:
why are we returning a string in the catch caluse? Also `str` does not allow the usage of format specifiers. Please have a look at [Format specifiers in clojure](https://clojuredocs.org/clojure.core/format)

Reply:
Yes, it's required anymore. I was using it for one more check but found that it was handled already.